### DISCUSSION TITLE
como postar aqui?

### DISCUSSION THREAD

**Discussion by albanesere** [clube-de-leitura-python/pense-em-python-2021#19] [Mostre alguma coisa]
Eu estou estudando o que perdi, e como sou professor há tanto tempo (bons 40 anos) eu só sei estudar escrevendo uma aula. Só sei fazer no ppoint, e queria postar aqui para vcs verem como ficou. Não tem meus desenhos e animações para ganhar tempo, mas ta bem organizadinho. Eu também resolvi os exercícios no meu notebook e queria postar aqui, apesar que comi bola no exercicio um. Ficar velha é triste, né? não sei como fazer no python a conversão dos 159,13 segundos para minutos e segundos eu sei que são 2 minutos e 39 segundos... porque fiz a conta de cabeça... como faz no python? porque se dividie dá a bagunça da base 10 com a base 60...

**Comment by villares** [comment]
Salve Albanesere!

ótimas questões... pra postar imagens/slides, dependendo o tipo de arquivo você poderia subir no Google Drive ou DropBox e mandar o link aqui. Imagens menores (como capturas de tela, que eu uso muito) você pode dar um Colar/Paste aqui nas mensagens que ele sobe e incorpora.

A divisão eu fiz assim, 159 segundos por 60  ->   159 // 60 dá um valor em minutos "floor division", despreza a parte fracionária, devolve o valor 2, e você pode obter o resto da divisão com o operador % assim -> 159 % 60  (que são os 39 segundos)

**Comment by albanesere** [comment]
showwwwwww... eu fazia isso na sala de aula com as crianças e travei...
esqueci... que lindo
Só retomando... 159 % 60 é o comando no python pra saber o resto, certo?
Então, eu tenho que trabalhar, e parei os estudos.
Vou passar meu caderno python no meu drive do google e vou colocar o link
pra vcs
Se estiver algo errado, por favor, corrijam e me avisem.
Agradeço
E eu fiz tudo no python... hahaha... bonitinho... como o autor mandou...
tem os projetos salvos . Uso o anaconda no desktop, se quiser mando tudo no
drive.
Logo eu termino.
Abr

Agradeço muito por tudo!
Re

**Reply by villares** [reply]
Outra dica , se você quiser compartilhar código aqui você escreve três crases/backticks ` ``` `  no começo, seguido da palavra `python` e no final também, assim:

` ```python`
```
# seu código aqui
```
` ``` `

**Comment by albanesere** [comment]
entendi
Esses códigos que fiz são babacas... mas vou usar isso para os que eu
futuramente desenvolver com o livro
Agradeço
Re

Em seg., 30 de ago. de 2021 às 12:03, Alexandre B A Villares <
***@***.***> escreveu:

**Comment by albanesere** [comment]
Vou deixar então o link aqui dos arquivos que estou fazendo.
Ab
Re
https://drive.google.com/drive/folders/1QxH5tnbfYcgx1rklWTG4GHIbRR6gOdIw?usp=sharing

**Comment by albanesere** [comment]
Por favor qualquer coisa que esteja errada peço que informem, eu deixei travado para não me atrapalhar em minha organização, mas se algum slide estiver errado peço que me informem por favor.
Agradeço.
Ab
Re
